{"repository":"https://github.com/neuro-inc/neuro-flow","dockerfilePath":"Dockerfile","startTime":1672725221584,"clone":{"stdout":"","stderr":"Cloning into '/tmp/dinghy-analysis/neuro-inc/neuro-flow'...\n","error":null,"commit":"8da7c8d57edfc5f6f7d0571d19c8d56796e73abb"},"originalBuild":{"startTimestamp":1672725222623,"endTimestamp":1672725255567,"error":{"code":1,"killed":false,"signal":null,"cmd":"docker build --no-cache --force-rm -t neuro-flow:latest -f /tmp/dinghy-analysis/neuro-inc/neuro-flow/Dockerfile ."},"stdout":"Sending build context to Docker daemon  4.608kB\r\r\nStep 1/14 : FROM python:3.11.1-buster as requirements\n3.11.1-buster: Pulling from library/python\nc7c50787e2da: Already exists\naff359114acb: Already exists\n821dc261e045: Already exists\n3790459d63d5: Already exists\nc09b016fff62: Already exists\n6aa81cf5366e: Pulling fs layer\n32cbf5071a0f: Pulling fs layer\n378902bafcb8: Pulling fs layer\n481826b0920f: Pulling fs layer\n481826b0920f: Waiting\n378902bafcb8: Download complete\n6aa81cf5366e: Download complete\n32cbf5071a0f: Verifying Checksum\n32cbf5071a0f: Download complete\n481826b0920f: Verifying Checksum\n481826b0920f: Download complete\n6aa81cf5366e: Pull complete\n32cbf5071a0f: Pull complete\n378902bafcb8: Pull complete\n481826b0920f: Pull complete\nDigest: sha256:86699a322b6cfcb6383448e853af68dbe07bf20a20301a5e9ef84d1479c76584\nStatus: Downloaded newer image for python:3.11.1-buster\n ---> 8ec09d49ecb9\nStep 2/14 : ARG NEURO_FLOW_DIST\n ---> Running in 554a0e624a38\nRemoving intermediate container 554a0e624a38\n ---> d04d28f1bcfd\nStep 3/14 : ENV PATH=/root/.local/bin:$PATH\n ---> Running in a3cc4d16f19c\nRemoving intermediate container a3cc4d16f19c\n ---> 189aab88fe1f\nStep 4/14 : RUN pip install --user --upgrade pip\n ---> Running in 2af84ea2c368\nRequirement already satisfied: pip in /usr/local/lib/python3.11/site-packages (22.3.1)\n\u001b[91mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\n\u001b[0mRemoving intermediate container 2af84ea2c368\n ---> 45c08efb25ff\nStep 5/14 : ADD ./dist /dist\n","stderr":"ADD failed: file not found in build context or excluded by .dockerignore: stat dist: file does not exist\n"},"endTime":1672725255636}